controllers/oauth_token_controller.go

```diff
@@ -32,7 +32,9 @@ func (otc *OAuthTokenController) Create(clientId, clientSecret, accessCode strin
 		Execute()
 
 	if err != nil {
-		return nil, err
+		if _, ok := err.(wrecker.ResponseError); !ok {
+			return nil, err
+		}
```

Comment:
This is a change required for the upgrade to `Wrecker v0.1.4`.  Wrecker now returns an error in the event of an HTTP error.  We need to ignore this, as we are performing the check below.